Makefile

```diff
@@ -30,6 +33,24 @@ tests:
 test:
 	(cd "${DIR}"; dotnet test Source/IntegrationTests --filter "DisplayName~${name}")
 
+# Run Dafny on an integration test case directly in the folder itself.
+# make test-run name=<part of the path> action="run ..."
+test-dafny:
+	name="$(name)"; \
+	files=$$(cd "${DIR}"/Source/IntegrationTests/TestFiles/LitTests/LitTest; find . -type f -wholename "*$$name*" | grep -E '\.dfy$$'); \
+	count=$$(echo "$$files" | wc -l); \
+  echo "$${files}"; \
+	if [ "$$count" -eq 0 ]; then \
+		echo "No files found matching pattern: $$name"; \
+		exit 1; \
+	else \
+		echo "$$count test files found."; \
+		for file in $$files; do \
+			filedir=$$(dirname "$$file"); \
+			(cd "${DIR}/Source/IntegrationTests/TestFiles/LitTests/LitTest/$${filedir}"; "${DIR}"/Binaries/Dafny $(action)  "$$(basename $$file)" ); \
```

Comment:
Not blocking since this doesn't appear to be used for any automation, but I would suggest using `dotnet run ...` instead of `Binaries/Dafny` to prevent the mistake of running `make test-run ...` on an outdated binary.